core/src/test/scala/unit/kafka/server/OffsetCommitTest.scala

```diff
@@ -249,11 +249,11 @@ class OffsetCommitTest extends ZooKeeperTestHarness {
     Thread.sleep(retentionCheckInterval * 2)
     assertEquals(2L, simpleConsumer.fetchOffsets(fetchRequest).requestInfo.get(topicPartition).get.offset)
 
-    // v1 version commit request with commit timestamp set to now - two days
+    // v1 version commit request with commit timestamp set to now - eight days
     // committed offset should expire
     val commitRequest2 = OffsetCommitRequest(
       groupId = group,
-      requestInfo = immutable.Map(topicPartition -> OffsetAndMetadata(3L, "metadata", Time.SYSTEM.milliseconds - 2*24*60*60*1000L)),
+      requestInfo = immutable.Map(topicPartition -> OffsetAndMetadata(3L, "metadata", Time.SYSTEM.milliseconds - 8*24*60*60*1000L)),
```

Comment:
I don't suppose we could refer to the default value directly?

Reply:
Happy to change it. I'm never sure with tests like these. On the one hand, using the variable directly seems cleaner and easier to maintain. On the other hand, using it directly means a regression like accidentally changing it would not be caught here.

Reply:
Ha, I didn't consider that. This test case doesn't seem to be targeting the default expiration specifically anyway, but I take the point.